packages/core/karma.conf.cjs

```diff
@@ -20,7 +20,17 @@ module.exports = function (config) {
     webpack: {
       mode: "development",
       module: {
-        rules: [{ test: /\.([cm]?ts|tsx)$/, loader: "ts-loader" }]
+        rules: [
+          { 
+            test: /\.([cm]?ts|tsx)$/,
+            loader: "ts-loader"
+          }, {
+            test: /\.m?js/,
```

Comment:
`webpack`'s build was failing due to still resolving with `.js`
https://github.com/webpack/webpack/issues/11467#issuecomment-691873586